src/stringlist-to-array.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+// Converts a go-ipfs "stringList" to an array
+// { Strings: ['A', 'B'] } --> ['A', 'B']
+function stringlistToArray (res, cb) {
+  cb(null, res.Strings || [])
+}
+
+module.exports = stringlistToArray
```

Comment:
What is a string list? Is there a better name in this context?  Does it has to be a separate file to do this?